service/controller/clusterapi/machine_deployment.go

```diff
@@ -0,0 +1,147 @@
+package clusterapi
+
+import (
+	clusterv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/cluster/v1alpha1"
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/giantswarm/operatorkit/client/k8scrdclient"
+	"github.com/giantswarm/operatorkit/controller"
+	"github.com/giantswarm/operatorkit/informer"
+	corev1 "k8s.io/api/core/v1"
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/cluster-api/pkg/client/clientset_generated/clientset"
+
+	"github.com/giantswarm/aws-operator/client/aws"
+	"github.com/giantswarm/aws-operator/service/controller/clusterapi/v28"
+)
+
+type MachineDeploymentConfig struct {
+	CMAClient    clientset.Interface
+	G8sClient    versioned.Interface
+	K8sClient    kubernetes.Interface
+	K8sExtClient apiextensionsclient.Interface
+	Logger       micrologger.Logger
+
+	HostAWSConfig  aws.Config
+	ProjectName    string
+	Route53Enabled bool
+}
+
+type MachineDeployment struct {
+	*controller.Controller
+}
+
+func NewMachineDeployment(config MachineDeploymentConfig) (*MachineDeployment, error) {
```

Comment:
Here is the new controller. The only thing we actually want to achieve for now is that the CRD is installed via operatorkit.